src/styles/NewsCard.css

```diff
@@ -0,0 +1,41 @@
+.news-card {
+  width: 100%;
+  padding: 8px 8px;
+  align-items: center;
+  justify-content: center;
+  border-radius: 4px;
+  background-color: white;
+  border: solid 1px white;
+}
+
+.news-card__data-and-tag {
+  display: flex;
+  padding: 9px 0px;
+}
+
+.news-card__data-and-tag__date {
+  font: bold;
```

Comment:
[imo]
この書き方でも反映はされた気がしますが、明示的に `font-weight: bold;` としてあげる方が、より良いと思いました。

```suggestion
  font-weight: bold;
```

<img width="1014" alt="image" src="https://user-images.githubusercontent.com/12077457/171530958-795a7400-b945-41c7-9a82-0c1505098309.png">